### DISCUSSION TITLE
Route only Grounds on Bottom Layer (2 Layer Board) in Kicad 8

### DISCUSSION THREAD

**Discussion by SidSkiba** [freerouting/freerouting#327] [Q&A]
Hoping someone may be able to point me. I have tried a number of things and cannot figure out if I can have Freerouting route everything on the top layer (except grounds) and route grounds only on the bottom layer. Or if that is even possible. I have tried using net classes but they do not seem to have any impact. Freerouting places tracks top and bottom regardless. 

Thx.

**Comment by andrasfuchs** [comment]
I'm not sure if it's possible with it, but have you tried our `-inc` command line argument to set the ignored net classes?

```
-inc [net class names, separated by commas]: auto-router ignores the listed net classes, eg. -inc GND,VCC will not try to wire components that are either in the "GND" or in the "VCC" net class.
```
